pkg/providers/aws/credentials.go

```diff
@@ -42,6 +42,7 @@ var (
 				"s3:PutBucketTagging",
 				"s3:PutBucketPublicAccessBlock",
 				"s3:PutEncryptionConfiguration",
+				"s3:PutBucketPolicy",
```

Comment:
When adding new permissions to this list, you need to update this document too https://docs.google.com/document/d/11UwspEIuOvowWoILaI_3orQdBdDX_geCx9YABUmL3Kc/edit?usp=sharing

Reply:
Will update if we get confirmation from BU, thanks!